init/initialize.sh

```diff
@@ -97,6 +97,15 @@ sed -e "s;RSYNCDIR_SS;$RSYNCDIR_SS;" \
     -e "s;RSYNCDIR_NPAD;$RSYNCDIR_NPAD;" \
     -e "s;RSYNCDIR_PTR;$RSYNCDIR_PTR;" \
     $SLICEHOME/conf/rsyncd.conf.in > /etc/rsyncd.conf
+
+# Allow CIRA rsync access to the paris-traceroute module at their own sites
+for site in "yyz01 yyc01 yul01"; do
+    if echo $(hostname) | grep -q $site; then
```

Comment:
I had considered using bash's regex matching operator here, but decided that echo and grep are universal and easy to read. However, using this syntax looks cleaner, and since we probably have bashisms all over the place anyway, there is likely no point in trying to make the scripts shell-agnostic.  I've added a commit which changes this.